src/Seq.Input.HealthCheck/HttpHealthCheck.cs

```diff
@@ -40,15 +43,32 @@ class HttpHealthCheck
         const int InitialContentChars = 16;
         const string OutcomeSucceeded = "succeeded", OutcomeFailed = "failed";
 
-        public HttpHealthCheck(HttpClient httpClient, string title, string targetUrl, JsonDataExtractor extractor, bool bypassHttpCaching)
+        public HttpHealthCheck(HttpClient httpClient, string title, string targetUrl, string authenticationHeaderValue, string optionalHeader, JsonDataExtractor extractor, bool bypassHttpCaching)
```

Comment:
The authentication header should be a `Name: Value` pair like the others, it may be e.g. `X-Some-ApiKey`, not necessarily just `Authorization`.

Reply:
I'm not sure how to resolve this here.
It's sensible that the UI has a specific `Authorization` password field, as I wouldn't want to leak keys or passwords in UI.

I can either glue both values together, have another optional field to change the authorization attribute name, and/or combine the optionalHeaders with the authorization headers before the constructor.

Thanks for you advice!